azure/Guidelines.md

```diff
@@ -1020,28 +1020,40 @@ Depending on the requirements of the service, there can be any number of "input"
 
 <a href="#condreq" name="condreq"></a>
 ### Conditional Requests
-When designing an API, you will almost certainly have to manage how your resource is updated. For example, if your resource is a bank account, you will want to ensure that one transaction--say depositing money--does not overwrite a previous transaction.
-Similarly, it could be very expensive to send a resource to a client. This could be because of its size, network conditions, or a myriad of other reasons. To enable this level of control, services should leverage an `ETag` header, or "entity tag," which will identify the 'version' or 'instance' of the resource a particular client is working with.
-An `ETag` is always set by the service and will enable you to _conditionally_ control how your service responds to requests, enabling you to provide predictable updates and more efficient access.
 
-<a href="#condreq-return-etags" name="condreq-return-etags">:ballot_box_with_check:</a> **YOU SHOULD** return an `ETag` with any operation returning the resource or part of a resource or any update of the resource (whether the resource is returned or not).
+The [HTTP Standard][] defines request headers that clients may use to specify a _precondition_
```

Comment:
Tip: at least with GitHub-flavored markdown, if you use an anchor like this you don't need the empty brackets i.e.,
```suggestion
The [HTTP Standard] defines request headers that clients may use to specify a _precondition_
```